packages/svelte/package.json

```diff
@@ -19,17 +20,54 @@
   "keywords": [
     "svelte"
   ],
-  "exports": {
-    ".": "./src/index.js",
-    "./server": "./src/server.js"
+  "scripts": {
+    "build": "svelte-kit sync && svelte-package && publint",
+    "prepublishOnly": "npm run build",
+    "check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+    "check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch"
   },
-  "main": "src/index.js",
+  "files": [
+    "dist",
+    "!dist/**/*.test.*",
+    "!dist/**/*.spec.*"
+  ],
   "peerDependencies": {
     "svelte": "^3.20.0 || ^4.0.0"
   },
+  "devDependencies": {
+    "@sveltejs/adapter-auto": "^3.2.0",
+    "@sveltejs/kit": "^2.5.5",
+    "@sveltejs/package": "^2.3.0",
+    "@sveltejs/vite-plugin-svelte": "^3.0.2",
+    "publint": "^0.2.7",
+    "svelte": "^4.2.12",
+    "svelte-check": "^3.6.8",
+    "tslib": "^2.6.2",
+    "typescript": "^5.4.3",
+    "vite": "^5.2.7"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "svelte": "./dist/index.js"
+    },
+    "./server": {
+      "types": "./dist/server.d.ts",
+      "svelte": "./dist/server.js"
+    }
+  },
+  "import": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "type": "module",
+  "typesVersions": {
+    ">4.0": {
+      ".": [
+        "./dist/index.d.ts"
+      ]
+    }
+  },
   "dependencies": {
-    "@inertiajs/core": "1.0.16",
-    "lodash.clonedeep": "^4.5.0",
-    "lodash.isequal": "^4.5.0"
+    "@inertiajs/core": "^1.0.16",
+    "lodash": "^4.5.0"
```

Comment:
These days we shouldn't rely on lodash anymore as it is no longer maintained. Importing the whole library now is also worst than importing the 2 small one.

I believe "lodash.clonedeep" could be replaced by the native structuredClone https://developer.mozilla.org/en-US/docs/Web/API/structuredClone

Another alternative would be switching from lodash to Remeda

cloneDeep: https://remedajs.com/docs/#clone
deepEqual: https://remedajs.com/docs/#isDeepEqual

This lib is maintained and free of vulnerabilities

Reply:
The whole library is [not imported](https://github.com/inertiajs/inertia/pull/1866/files#diff-42290667298535b040cc6d17c720543abdc4c74ed4e30d1276ac827dcbe7aaa0R3-R4), so there is no extra bundle size usage, however using 1 package instead of 2 is a bit cleaner.

Switching to other library or ditching it as a whole should be consideration for other PR.